modules/ixBidAdapter.js

```diff
@@ -285,6 +299,14 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
   r.ext.source = 'prebid';
   r.ext.ixdiag = {};
 
+  // getting ixdiags for adunits of the video, outstream & multi format (MF) style
+  let ixdiagMF = buildMultiFormatDiag(validBidRequests);
+  for (var key in ixdiagMF) {
+    if (ixdiagMF.hasOwnProperty(key)) {
```

Comment:
Why is this check needed? What properties will be in `ixdiagMF` with `hasOwnProperty` returning `false`?